library/core/class.form.php

```diff
@@ -157,15 +157,27 @@ public function BodyBox($Column = 'Body', $Attributes = array()) {
       $Attributes['format'] = htmlspecialchars($Attributes['format']);
       $this->SetValue('Format', $Attributes['format']);
```

Comment:
Can you instead move this after the event and check for an empty string or something? The `IncludeFormat` just seems too kludgey.